.gitlab/deps_build/deps_build.yml

```diff
@@ -122,3 +122,41 @@ build_processed_btfhub_archive:
     - dda inv -- -e system-probe.process-btfhub-archive --branch $BTFHUB_ARCHIVE_BRANCH
```

Comment:
I would rather have a separate yaml file to include it to keep the current `deps_build.yml` clean. Something like `deps_bazel_build.yml`